README.md

```diff
@@ -1,3 +1,9 @@
+# Mattermost/GitLab Integration
```

Comment:
Suggestion: This can likely be removed as this PR adds content that fulfills the stated note.

Reply:
I was thinking about that.  However, two things occurred in my mind:

- When I looked at the JIRA plug-in it still had the README.md file and pointed to the new Gitbook.  I figured I needed such a pointer from the default README.md that everyone looks at.

- The reason I kept the entire README.md content, instead of deleting it and pointing to the Gitbook, is that firstly the Jira README.md did that and I did not know if there was a phase out period we wanted to honor.  The other reason is that I don't really know what the process is for this Gitbook to be deployed as https://mattermost.gitbook.io/plugin-gitlab.  I was looking in the makefile for the Jira plugin if there was something that generated that  formalized that content.  I did not see anything obvious.  I was not sure if that is a manual process or does it just happen auto-magically.  Either way, we would need that to be in place before we could eliminate the content in the README.md file and replace it with a link to the newly deployed https://mattermost.gitbook.io/plugin-jira Gitbook.

Reply:
Those are excellent points, thanks. I'm  going to defer to @aaronrothschild, especially around plans for this content going forward and deployment.

Reply:
Thanks for this, @RohitJain13. An update on this thread: The README.md file will be replaced with a link to the Gitbook for this documentation, and will also include links to the developer documentation and admin documentation. This will be implemented in the next few weeks. :)